### PR TITLE
Create data model

### PR DESCRIPTION
**Pull request by mwenyoa**
### Project requirements
Be sure to reference the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) from the sneak peek if you have any questions about table or column names or data relationships.

- Create a new app. Give it a meaningful name.
- Initialize your project with Git.
- Make sure that your project has Postgres database set up.
- Build your project schema.
    - Create and run the necessary migration files.
    - Table and column names should match the ERD diagram.
        - note: photo for users table should be a link to an image
    - Foreign keys should be included.
    - All columns that are foreign keys should have a corresponding index.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+# It's strongly recommended that you check this file into your version control system.
+
+# rubocop:disable Metrics/BlockLength
+ActiveRecord::Schema[7.0].define(version: 20_220_810_124_313) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension 'plpgsql'
+
+  create_table 'comments', force: :cascade do |t|
+    t.integer 'authorid'
+    t.integer 'postid'
```
Comment (julie-ify) [inline db/schema.rb]: - [ ] We need to keep our tables clean and remove unnecessary attributes. Kindly run new migrations to remove these `postid` and `authorid` from your comments table. You already have `post_id` and `author_id`, and those are enough.

File: db/schema.rb [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    t.bigint 'post_id', null: false
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+    t.index ['author_id'], name: 'index_comments_on_author_id'
+    t.index ['post_id'], name: 'index_comments_on_post_id'
+  end
+
+  create_table 'likes', force: :cascade do |t|
+    t.integer 'authorid'
+    t.integer 'postid'
```
Comment (julie-ify) [inline db/schema.rb]: - [ ] Kindly remove these unneeded attributes from the likes table.

File: db/schema.rb [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    t.bigint 'author_id', null: false
+    t.bigint 'post_id', null: false
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+    t.index ['author_id'], name: 'index_likes_on_author_id'
+    t.index ['post_id'], name: 'index_likes_on_post_id'
+  end
+
+  create_table 'posts', force: :cascade do |t|
+    t.integer 'authorid'
```
Comment (julie-ify) [inline db/schema.rb]: - [ ] In the posts table we already have `author_id`, kindly remove this one here as it unnecessary.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @mwenyoa,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/schema.rb (hunk 3 above)
Comment (mwenyoa) [reply db/schema.rb]: Hi @julie-ify Thanks for your feedback, the issues highlighted have now been worked on.

**Review by FrederickMih — APPROVED** [review]
Hi @mwenyoa, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 





Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mwenyoa** [comment]
Hi @FrederickMih ,  @julie-ify thanks alot for taking time to review my code.

**Pull request merged**